datadog_checks_dev/datadog_checks/dev/tooling/templates/docs/setup/configuration/log_collection.yaml

```diff
@@ -0,0 +1,28 @@
+name: Log Collection
+header_level: 4
+description: |
+  _Available for Agent versions >6.0_
+
+  1. Collecting logs is disabled by default in the Datadog Agent, enable it in your `datadog.yaml` file:
+
+    ```yaml
+    logs_enabled: true
+    ```
```

Comment:
I think this needs to be indented 2 extra spaces? Same for `check_log_conf_snippet`, see RethinkDB in prod: https://docs.datadoghq.com/integrations/rethinkdb